RestApi/Sepes.RestApi/Startup.cs

```diff
@@ -79,7 +79,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
             app.UseCors(MyAllowSpecificOrigins);
-            app.UseHttpsRedirection();
+
+            // UseHttpsRedirection doesn't work well with docker.
+            if(env.EnvironmentName == "Development") {
```

Comment:
Would it not be preferable to set environment name as "docker-prod" in the compose file and check against that?
As HTTPS is a requirement for this apps security and docker should be an edge case exception

Reply:
Have new solution with a SEPES_HTTP_ONLY setting for when behind a proxy.